spanner/changestreams/src/main/java/com/example/spanner/changestreams/ChangeStreamSample.java

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.changestreams;
+
+// TODO(haikuo@google.com): we should remove these models once the change stream connector code that
+// includes these models are released into Beam repo.
+import com.example.spanner.changestreams.model.ChangeStreamRecord;
+import com.example.spanner.changestreams.model.ChildPartition;
+import com.example.spanner.changestreams.model.ChildPartitionsRecord;
+import com.example.spanner.changestreams.model.DataChangeRecord;
+import com.example.spanner.changestreams.model.HeartbeatRecord;
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.Timestamp;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.common.collect.ImmutableList;
+import com.google.spanner.admin.database.v1.UpdateDatabaseDdlMetadata;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Sample code for querying change stream, it:
+ * 1. Creates a table with simple schema and a change stream that watches the table.
+ * 2. Inserts test data into the table.
+ * 3. Executes a change stream initial query to get change stream partition tokens.
+ * 4. Executes a change stream partition query to get data change records of the inserted rows.
+ * 5. Drops the created table and change stream.
+ */
+public class ChangeStreamSample {
+  private static final long TIMEOUT_MINUTES = 2;
+
+  public static void main(String... args) {
+    if (args.length != 3) {
+      System.err.println("Usage: ChangeStreamSample "
+          + "<instance_id> "
+          + "<database_id> "
+          + "<prefix>");
+      return;
+    }
+
+    final String instanceId = args[0];
+    final String databaseId = args[1];
+    final String prefix = args[2];
+    final String tableName = prefix + "_Singers";
+    final String changeStreamName = prefix + "_ChangeStreamSingers";
+    SpannerOptions options = SpannerOptions.newBuilder().build();
```

Comment:
nit: could we make the other variables `final` as well, since you are using `final` above?

Reply:
Done.